aws-devicefarm-gradle-plugin/src/main/groovy/com.amazonaws.devicefarm/DeviceFarmServer.java

```diff
@@ -48,6 +50,20 @@
     private final DeviceFarmUploader uploader;
     private final DeviceFarmUtils utils;
 
+
+    /**
+     * Check whether the input test type is Appium.
+     *
+     * @param testType
+     * @return ture if it is an Appium test
+     */
+    private static boolean checkIfAppium(final TestType testType) {
```

Comment:
Normally boolean check methods like this follow the `isXXX` pattern.

"checkIfAppium" -> "isAppiumTestType"